### ISSUE TITLE
Add tests for text clipping

### ISSUE THREAD

**Issue by kittenchilly** [rh-hideout/pokeemerald-expansion#4231]
### Description

Instead of automatic text wrapping, I think we should use similar code for a test to check if any text clips.
So we can just do it manually, which would look better.

### Discord contact info

kittenchilly

**Comment by AsparagusEduardo** [comment]
@mrgriffin would you say this has been handled by #4241 at this point?

**Comment by mrgriffin** [comment]
I think kittenchilly intended for this issue to cover messages in battles. [Recent Discord discussion](https://discord.com/channels/419213663107416084/1077009799293710357/1264608402785697845).

**Comment by kittenchilly** [comment]
Yes, this was intended for battle messages, should've made that clearer

**Comment by Bassoonian** [comment]
Can this be marked as solved now that ALB is a thing?

**Comment by hedara90** [comment]
If it's only about battle messages, yes.

**Comment by Bassoonian** [comment]
Everything else has since received tests

**Comment by hedara90** [comment]
I'd say this is solved then. It's not strictly a test, but battle strings won't clip, and if they do it's a bug with the implementation.
